results/bert-base-multilingual-cased/FloresBitextMining.json

```diff
@@ -0,0 +1,22 @@
+{
+  "dataset_revision": "80dc3040d19756742c9a18267ab30f54fb8e226b",
+  "dev": {
```

Comment:
I'm not sure but I think the key is "test" here the other datasets, so it won't be a problem afterwards?

Reply:
Actually, Flores doesn't have a `test` split, it is named `dev` in HF (cf. https://huggingface.co/datasets/facebook/flores/blob/main/flores.py). 

@GabrielSequeira can we replace the split name with `test` in `mteb-french` ?

Reply:
I'm creating an issue for this in `mteb-french`:  https://github.com/Lyon-NLP/mteb-french/issues/43